modules/Microsoft.Insights/webTests/.test/min/deploy.test.bicep

```diff
@@ -48,6 +48,8 @@ module testDeployment '../../deploy.bicep' = {
   params: {
     name: '<<namePrefix>>${serviceShort}001'
     tags: {
+      Environment: 'Non-Prod'
+      Role: 'DeploymentValidation'
```

Comment:
Hey @shaban-khan please bare with me and remove this tags from the min test. They were part of a suggestion done before knowing of the mandatory hidden-link tag
```suggestion
```